inc/css-builder.php

```diff
@@ -217,9 +217,11 @@ public function get_css() {
 				continue;
 			}
 
-			if ( $max_res < 1920 ) {
+			if ( empty( $max_res ) && ! empty( $min_res ) ) {
```

Comment:
We need to specifically check for `$max_res === ''` because `0` is also considered an empty value.

Reply:
That was actually intentional on my part (as I think `0` instead of nothing is neater), but if you would rather not allow `0` at all that's fine - I'll make the change now.

Reply:
`0` is a valid number and should be allowed as a value, but it doesn't mean the same as 'not set'.

Reply:
Just to confirm, no further action is required, right? I ask as the last commit checks for`$max_res === ''`

Reply:
Ideally, this would be `if ( $max_res === '' && $min_res > 0 )` as that's more precisely what we're trying to check here, but I think this will work as is.